services/github/github-manifest.service.js

```diff
@@ -0,0 +1,159 @@
+'use strict'
+
+const Joi = require('joi')
+const { renderVersionBadge } = require('../../lib/version')
+const {
+  individualValueSchema,
+  transformAndValidate,
+  renderDynamicBadge,
+} = require('../dynamic-common')
+const { ConditionalGithubAuthService } = require('./github-auth-service')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+
+const schema = Joi.object({
+  version: individualValueSchema,
+}).required()
+
+class GithubManifestVersion extends ConditionalGithubAuthService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'github/manifest-json/v',
+      format: '([^/]+)/([^/]+)/?([^/]+)?',
+      capture: ['user', 'repo', 'branch'],
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'GitHub manifest version',
+        pattern: 'manifest-json/v/:user/:repo',
```

Comment:
Will the `manifest-json/v` portion be repeated here since that's defined in the route `base`?

Reply:
Oops! Indeed. Fixing that.

You can see it in the review app here: https://shields-staging-pr-2470.herokuapp.com/#/